components/merkledb/Cargo.toml

```
@@ -23,7 +23,9 @@ failure = "0.1"
 hex = "0.3.2"
 leb128 = "0.2"
 num-traits = "0.2"
-rocksdb = "0.12.0"
+
+# TODO: revert to original rocksdb
+rocksdb = { git = "https://github.com/pavel-mukhanov/rust-rocksdb", branch = "sync_snapshot" }
```

```suggestion
rocksdb = "0.12.3"
```
0.12.3 has been released !